akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -335,6 +348,14 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
         def enqueueOutStream(outStream: OutStream): Unit =
           if (!sendableOutstreams.contains(outStream.streamId))
             become(copy(sendableOutstreams = sendableOutstreams + outStream.streamId))
+
+        def closeStream(outStream: OutStream): Unit =
```

Comment:
Can you also add a test case for this scenario? You will have to exhaust the connection window before it ends up here but there should already be tests doing that.